.github/qa-sq-behind-ngix/generate-certificates.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+set -eux
+
+echo Generating server certificate...
+
+openssl req \
+    -newkey rsa:4096 \
+    -x509 \
+    -sha256 \
+    -addext "subjectAltName = DNS:localhost" \
+    -days 3650 \
+    -nodes \
+    -out server.crt \
+    -subj "/C=CH/ST=Geneva/L=Geneva/O=Server/OU=Dept" \
+    -keyout server.key
+
+echo Generating client certificate...
```

Comment:
I haven't seen any test configuring client certificates. Is this really used somewhere?
I don't think we need to test client certificates in this GH action, since we don't provide any special integration on top of the scanner CLI.

Reply:
Indeed, you're right, they were not necessary. I have removed them.

Reply:
It turns out that the CA certificate was not necessary either (as the server certificate is passed to both nginx and the CLI). I have removed that too.